pytaskflow/storage/redis_storage.py

```
@@ -68,37 +145,176 @@ def _serialize_job_for_storage(self, job: Job) -> dict:
         job_dict = {}
         for key, value in job_data.items():
-            # Ensure key is a string (decode if it's bytes)
-            if isinstance(key, bytes):
-                key = key.decode("utf-8")
-            # Ensure value is a string (decode if it's bytes)
-            if isinstance(value, bytes):
-                value = value.decode("utf-8")
-
-            if key == "created_at":
-                job_dict[key] = datetime.fromisoformat(value)
+            if isinstance(key, bytes): key = key.decode("utf-8")
```

Ruff (0.14.14)

[error] Multiple statements on one line (colon)

(E701)